pkg/descriptions.json

```diff
@@ -1975,6 +1975,13 @@
       "System.Text.Json.Utf8JsonReader"
     ]
   },
+  {
+    "Name": "Microsoft.Bcl.System.Text.Json.Sources",
+    "Description": "Provides the sources of System.Text.Json as internal types.",
```

Comment:
How about something a bit more descriptive like:

"Provides the source code implementation of System.Text.Json as internal types.  This package is only required for projects targeting .NET Framework 4.x or .NET Core 1.x/2.x; and is not for projects that can target .NET Core 3.0 and higher.

For more information on the Microsoft.Bcl.System.Text.Json.Sources package, visit GitHub: link_to_corefx_repo_md_file"

Reply:
Should that link to an md file be a link to main corefx repo md file? or one specific to the sources package? If the latter, then I would just be a bit worried to add that as a direct link as we may change that path in the future and won't be able to easily update shipped packages, so in that case I would rather have a forward link instead.